.github/workflows/ci.yml

```diff
@@ -199,29 +180,12 @@ jobs:
         brew uninstall -f bazel bazelisk
         brew update
         brew upgrade || true
-        brew install hdf5 git autoconf automake libtool curl wget pkg-config gsl numpy meson ninja
+        brew install hdf5 git libtool curl wget pkg-config gsl numpy meson ninja
```

Comment:
I suppose this is ok? Autoconf and automake not needed anymore? What about libtool, what is that used for?